test/integration/column_ordering_test.rb

```diff
@@ -20,4 +20,15 @@ def test_order_links_when_order_empty
     link = find("#table th.column-created_at a")
     assert_equal "/articles?order=created_at_desc", link[:href]
   end
+
+  def test_order_links_when_not_orderable
+    visit articles_path
+    assert has_no_link?("#table th.column-computed a")
```

Comment:
Oops. Forgot to change it :(
It passes because we check that the link does not exist, which it doesn't because I mistyped it :)

I changed it so now it checks that the selector exists, but it is not a link.